src/cmd/flux-wreck

```diff
@@ -141,6 +141,50 @@ prog:SubCommand {
  end
 }
 
+prog:SubCommand {
+ name = "exclude",
+ description = "Exclude a node from running future jobs",
+ usage = "NODENAME",
+ options = {
+   { name = "kill", char = "k",
+     usage = "kill the jobs to which the node is allocated"
+   }
+ },
+ handler = function (self, arg)
+    local function toboolean(X)
+         return not not X
+    end
+
+    local nodename = arg[1]
+    local resp, err = f:rpc ("sched.exclude", { node = nodename, kill = toboolean (self.opt.k) })
+    if not resp then
+        if err == "Function not implemented" and not self.opt.f then
+            prog:die ("Node exclusion is not supported when scheduler not loaded")
```

Comment:
I think `and not self.opt.f` will always be true here and could be removed (if I'm not mistaken). Same for the `include` command below.